.travis/before_install_test.sh

```diff
@@ -9,6 +9,20 @@ echo "memory_limit=3072M" >> "$TRAVIS_INI_FILE"
 
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+# TODO: remove when drop PHP 5 support
+# symfony/maker-bundle only works with PHP 7 and higher
+if [ "${TRAVIS_PHP_VERSION:0:3}" != "5.6" ]; then
+    # but only with Symfony 3.4 and higher
+    if [ "$SYMFONY" != "" ]; then
+        if [ "${SYMFONY:0:3}" != "2.8" ] && [ "${SYMFONY:0:3}" != "3.3" ] ; then
+            composer require "symfony/maker-bundle:$" --no-update
+        fi
+    else
+        composer require "symfony/maker-bundle:$" --no-update
+    fi
+fi
+
 if [ "$SYMFONY" != "" ]; then composer require "symfony/symfony:$SYMFONY" --no-update; fi;
 if [ "$SONATA_CORE" != "" ]; then composer require "sonata-project/core-bundle:$SONATA_CORE" --no-update; fi;
 if [ "$SONATA_BLOCK" != "" ]; then composer require "sonata-project/block-bundle:$SONATA_BLOCK" --no-update; fi;
+if [ "$SYMFONY_MAKER" != "" ]; then composer require "symfony/maker-bundle:$SYMFONY_MAKER" --no-update; fi;
```

Comment:
@sonata-project/contributors hmm this line shouldn't be there, because we have the block above.